### PR TITLE
Cans 1177 accessibility fix

### PR DESCRIPTION
**Pull request by abelayt**
## Tests

- [ ] I used TDD to develop the feature
- [x] I have included unit tests
- [ ] I have included new acceptance tests or modified existing ones
- [ ] I have included other tests

## Types of changes

- [ ] New feature (non-breaking change which adds functionality)
- [ ] Refactoring (No behavioral changes)
- [x] Bug fix (non-breaking change which fixes an issue)
- [ ] Breaking change (fix or feature that would cause existing functionality to not work as expected)

## Checklist:

- [x] My code follows the code style of this project.
- [x] I ran all my tests locally which were green.
- [x] My code adds no new issues to Code Climate
- [x] I ran all the linters for the project.
- [x] My code is in a stable state ready to be deployable, but not necessarily complete.
- [x] I promise on my honor as a CWDS developer to ensure I don't break the build (or bring donuts if I leave things broken), to check linters, use best practices, to leave the code in better shape than I found it, and to have a good day.

### CODE REVIEW

**Comment by abelayt** [comment]
retest this please

**Comment by abelayt** [comment]
retest this please

File: app/javascript/Application/components/Assessment/DomainScore.js [hunk 1]
```diff
@@ -6,7 +6,20 @@ import './style.sass'
...
+          whiteSpace: 'nowrap',
+          width: '5.5rem',
+          height: '1.125rem',
+          color: '#6F6F6F',
+          fontFamily: 'SourceSansPro - Regular',
+          fontWeight: 'normal',
+          lineHeight: '1.125rem',
+          marginRight: '0.2rem',
+        }}
+      >
```
Comment (AK56) [inline app/javascript/Application/components/Assessment/DomainScore.js]: Yeah the styles grew here, so we shall move them into the respective SAAS file

File: app/javascript/Application/components/Assessment/Domain.js [hunk 2]
```diff
@@ -3,7 +3,7 @@ import PropTypes from 'prop-types'
 import ExpansionPanel from '@material-ui/core/ExpansionPanel'
 import ExpansionPanelSummary from '@material-ui/core/ExpansionPanelSummary'
 import ExpansionPanelDetails from '@material-ui/core/ExpansionPanelDetails'
-import { Input } from 'reactstrap'
+import { Input, Label } from 'reactstrap'
```
Comment (denys-davydov) [inline app/javascript/Application/components/Assessment/Domain.js]: We should check if the Label component is available in the Component Library. Please try to do it or confirm it is not available.

Comment (abelayt) [reply app/javascript/Application/components/Assessment/Domain.js]: Thanks, now Label imported from @cwds/components

File: app/javascript/Application/components/Assessment/DomainScore.js (hunk 1 above)
Comment (abelayt) [reply app/javascript/Application/components/Assessment/DomainScore.js]: Thanks, moved it to .sass file.

**Pull request merged**